### DISCUSSION TITLE
link to open particular node in graph-explorer

### DISCUSSION THREAD

**Discussion by HareeshaSN** [aws/graph-explorer#576] [Q&A]
Hi,

is there a way to share a graph-explorer url with end user so that it will open particular node in graph-explorer?

End user is not technical person to navigate and find node which he is looking for so let say if we share graph-explorer url along with node details in the url so that graph-explorer directly opens that particular node graph when user takes that url.

**Comment by kmcginnes** [comment]
No, not yet. However, this has been a heavily requested feature. 

- #298
- #64
- #440

This is the next project we plan to tackle. There is no tentative release schedule for the work yet.
